components/dashboard/src/components/RepositoryFinder.tsx

```diff
@@ -100,7 +100,7 @@ export default function RepositoryFinder({
             const result = repos.map((repo) => {
                 return {
                     id: repo.projectId || repo.url,
-                    element: <SuggestedRepositoryOption repo={repo} />,
+                    element: <SuggestedRepositoryOption repo={repo} useProjectNameIfAvailable={!excludeProjects} />,
```

Comment:
I guess this removes names altogether?

<img width="489" alt="image" src="https://github.com/gitpod-io/gitpod/assets/29888641/8f3303bb-64d2-466b-b4c5-e82ce9ba3279">